src/logger.rs

```diff
@@ -110,6 +110,19 @@ impl Logger {
             StdoutLog::Off => {}
         }
 
+        logger.dispatch = logger
+            .dispatch
+            .level_for("gfx_backend_empty", LevelFilter::Warn)
+            .level_for("gfx_backend_vulkan", LevelFilter::Warn)
+            .level_for("gfx_backend_dx12", LevelFilter::Warn)
+            .level_for("gfx_backend_metal", LevelFilter::Warn)
+            .level_for("rendy_factory::factory", LevelFilter::Warn)
+            .level_for("rendy_memory::allocator::dynamic", LevelFilter::Warn)
+            .level_for("rendy_graph::node::render::pass", LevelFilter::Warn)
+            .level_for("rendy_graph::graph", LevelFilter::Warn)
+            .level_for("rendy_memory::allocator::linear", LevelFilter::Warn)
+            .level_for("rendy_wsi", LevelFilter::Warn);
+
         if let Some(log_gfx_device_level) = config.log_gfx_device_level {
```

Comment:
I believe this option `log_gfx_device_level` is outdated so it might be able to be removed, but could you put the rendering logging defaults into a config option like this one which defaults to Warn?